holidays/countries/russia.py

```diff
@@ -23,6 +23,11 @@ class Russia(HolidayBase):
     country = "RU"
     default_language = "ru"
 
+    special_holidays = {
+        2023: ((FEB, 24, "День защитника Отечества"),), # Bridge day for 1/Jan, src: https://www.consultant.ru/document/cons_doc_LAW_425407/
```

Comment:
Could you really run the pre-commit and make sure the code style looks good? Thank you!